app/harvesters/nara_harvester.rb

```diff
@@ -0,0 +1,126 @@
+##
+# A harvester for NARA's API
+#
+# @see Krikri::Harvesters::ApiHarvester
+class NaraHarvester < Krikri::Harvesters::ApiHarvester
+  DEFAULT_URI = 'https://catalog.archives.gov/api/v1'
+  DEFAULT_NAME = 'nara'
+  DEFAULT_BATCHSIZE = 10
+  DEFAULT_ID_FILENAME = '/var/tmp/nara_ids'
```

Comment:
Does this need to be hardcoded, or does this make more sense to be moved to configuration?